### PR TITLE
fix: storybook build (DX-1551)

### PR DESCRIPTION
**Pull request by ninabondar**
- fixes storybook build
- adds chromatic

### CODE REVIEW

File: packages/react-chat/.babelrc.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+          "safari": 15,
+          "firefox": 91
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```
Comment (ninabondar) [inline packages/react-chat/.babelrc.json:18]: this .babelrc was generated via `npx storybook@latest babelrc`

before adding this, got the same issue as featured below (link to the solution)
https://github.com/storybookjs/storybook/issues/22123#issuecomment-1514124086

Comment (ninabondar) [reply packages/react-chat/.babelrc.json]: I had node v16

**Comment by DecathectZero** [comment]
please make sure to test the built version of the webchat and ensure everything is working.

File: packages/react-chat/src/components/SystemResponse/SystemResponse.story.tsx [hunk 2]
```diff
@@ -29,128 +30,128 @@ export default {
...
-export const MultipleWithFeedback = Template.bind({});
-MultipleWithFeedback.args = {
-  feedback: {
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    onClick: () => {},
+export const MultipleWithFeedback: Story = {
+  args: {
+    feedback: {
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onClick: () => { },
```
Comment (ninabondar) [inline packages/react-chat/src/components/SystemResponse/SystemResponse.story.tsx:151]: `() => null`

File: packages/react-chat/.storybook/main.ts [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
+import { StorybookConfig } from '@storybook/react-vite';
 import { mergeConfig } from 'vite';
+import svgr from 'vite-plugin-svgr';
 
 import { createPlugins } from '../vite.config';
 
-const config: StorybookViteConfig = {
-  stories: ['../src/**/*.story.mdx', '../src/**/*.story.@(js|jsx|ts|tsx)', '../iframe/**/*.story.@(js|jsx|ts|tsx)'],
+const config: StorybookConfig = {
+  stories: ['../src/components/Button/Button.story.tsx'], // ['../src/**/*.story.@(js|jsx|ts|tsx)', '../iframe/**/*.story.@(js|jsx|ts|tsx)'],
```
Comment (ninabondar) [inline packages/react-chat/.storybook/main.ts:8]: OOPS need all stories

**Review by effervescentia — APPROVED** [review]

File: packages/react-chat/.storybook/main.ts [hunk 4]
```diff
@@ -27,11 +25,15 @@ const config: StorybookViteConfig = {
     },
   },
 
-  viteFinal: (config) =>
-    mergeConfig(config, {
-      base: '/react-chat/',
-      plugins: createPlugins(__dirname),
-    }),
+  viteFinal: (config) => {
+    console.log('config', config);
```
Comment (ninabondar) [inline packages/react-chat/.storybook/main.ts:29]: remove log

**Pull request merged**